app/src/main/java/com/github/sdpsharelook/storage/DatabaseViewFragment.kt

```diff
@@ -61,6 +60,12 @@ class DatabaseViewFragment : Fragment() {
         savedInstanceState: Bundle?
     ): View {
         _binding = FragmentDatabaseViewBinding.inflate(layoutInflater)
+        val frag = this
+        Dispatchers.IO.dispatch(Dispatchers.IO){
+            runBlocking {
+                frag.collectDBFlow()
+            }
+        }
```

Comment:
This is definitely not the place to do anything like that. But I'm done fighting, I'll just have to fix it later.